src/library-authoring/library-team/messages.ts

```diff
@@ -0,0 +1,163 @@
+import { defineMessages as _defineMessages } from '@edx/frontend-platform/i18n';
+import type { defineMessages as defineMessagesType } from 'react-intl';
+
+// frontend-platform currently doesn't provide types... do it ourselves.
+const defineMessages = _defineMessages as typeof defineMessagesType;
```

Comment:
```suggestion
import { defineMessages } from '@edx/frontend-platform/i18n';
```
Because I added https://github.com/openedx/frontend-app-authoring/blob/master/src/frontend-platform.d.ts you don't need this anymore.